bluepysnap/_plotting.py

```diff
@@ -133,8 +133,9 @@ def _update_raster_properties():
 
     report = filtered_report.report
 
-    dtype = spike_report[population_names[0]].nodes.property_dtypes[y_axis] if y_axis else None
-    if dtype and pd.api.types.is_categorical_dtype(dtype):
+    # use np.int64 if displaying node_ids
+    dtype = spike_report[population_names[0]].nodes.property_dtypes[y_axis] if y_axis else IDS_DTYPE
```

Comment:
Using `None` as dtype here was causing bluepy functional tests to fail because of this warning from pandas:
> FutureWarning: The default dtype for empty Series will be 'object' instead of 'float64' in a future version. Specify a dtype explicitly to silence this warning.

Reply:
makes sense